Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -499,12 +535,16 @@ void AsyncScrollingCoordinator::scheduleRenderingUpdate()
 LocalFrameView* AsyncScrollingCoordinator::frameViewForScrollingNode(LocalFrame& rootFrame, ScrollingNodeID scrollingNodeID) const
 {
     ASSERT(rootFrame.isRootFrame());
-    if (scrollingNodeID == m_scrollingStateTree->rootStateNode()->scrollingNodeID()) {
+    auto it = m_scrollingStateTrees.find(rootFrame.frameID());
```

Comment:
Maybe add `existingScrollingStateTreeForRootFrameID()` and use it here.

Reply:
Fixed.